### PR TITLE
Add triple notch info

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: common/source/docs/common-imu-notch-filtering.rst [hunk 1]
```diff
@@ -46,14 +46,14 @@ Checking Notch Filter Effectiveness
...
 
-Double-Notch
-============
+Double/Triple-Notch
+===================
 
-The software notch filters used are very "spikey" being relatively narrow but good at attenuation at their center. On larger copters the noise profile of the motors is quite dirty covering a broader range of frequencies than can be covered by a single notch filter. In order to address this situation it is possible to configure the harmonic notches as double notches that gives a wider spread of significant attenuation. To utilize this feature set :ref:`INS_HNTCH_OPTS <INS_HNTCH_OPTS>` to "1".
+The software notch filters used are very "spikey" being relatively narrow but good at attenuation at their center. On larger copters the noise profile of the motors is quite dirty covering a broader range of frequencies than can be covered by a single notch filter. In order to address this situation it is possible to configure the harmonic notches as double or triple notches that gives a wider spread of significant attenuation. To utilize this feature set :ref:`INS_HNTCH_OPTS <INS_HNTCH_OPTS>` to "1" for double notches, to "4" for triple notches.
```
Comment (andyp1per) [inline common/source/docs/common-imu-notch-filtering.rst:52]: It's bit 4 so the actual value would be 16

File: common/source/docs/common-imu-notch-filtering.rst [hunk 2]
```diff
@@ -46,14 +46,14 @@ Checking Notch Filter Effectiveness
...
 
-.. note:: Each notch has some CPU cost so if you configure both dynamic harmonics and double notches (:ref:`INS_HNTCH_OPTS <INS_HNTCH_OPTS>` set to 3) you will end up with 8 notches on your aircraft per IMU. On flight controllers with 3 IMUs, this totals 24 notches which is computationally significant and could impact operation. For example, with F4 cpus with one IMU, using :ref:`INS_GYRO_RATE<INS_GYRO_RATE>` =0 (1khz) this is safe, as is 3 IMUs running with fast sampling (:ref:`INS_GYRO_RATE<INS_GYRO_RATE>` =1 (2khz) on H7 cpus.
+.. note:: Each notch has some CPU cost so if you configure both dynamic harmonics and double/triple notches (:ref:`INS_HNTCH_OPTS <INS_HNTCH_OPTS>` set to 3 or 6) you will end up with many notches on your aircraft per IMU. On flight controllers with 3 IMUs, this totals 24 or more notches which is computationally significant and could impact operation. For example, with F4 cpus with one IMU, using :ref:`INS_GYRO_RATE<INS_GYRO_RATE>` =0 (1khz) this is safe, as is 3 IMUs running with fast sampling (:ref:`INS_GYRO_RATE<INS_GYRO_RATE>` =1 (2khz) on H7 cpus. Use triple notches with caution to computational loading.
```
Comment (andyp1per) [inline common/source/docs/common-imu-notch-filtering.rst:54]: This would be 3 or 18

File: common/source/docs/common-imu-notch-filtering.rst [hunk 3]
```diff
@@ -46,14 +46,14 @@ Checking Notch Filter Effectiveness
...
 
-Note also that with a double-notch the maximum attenuation is either side of the center frequency so on smaller aircraft with a very pronounced peak their use is usually counter productive.
+Note also that with a double/triple-notch, the maximum attenuation is either side of the center frequency so on smaller aircraft with a very pronounced peak their use is usually counter productive.
```
Comment (andyp1per) [inline common/source/docs/common-imu-notch-filtering.rst:56]: This only applies to the double notch

**Review by andyp1per — CHANGES_REQUESTED** [review]

**Comment by Hwurzburg** [comment]
@andyp1per thanks for the review...fixed...

**Pull request merged**